### ISSUE TITLE
Platform ble_monitor does not generate unique IDs

### ISSUE THREAD

**Issue by eduperez** [custom-components/remote_homeassistant#76]
I have several BLE sensors scattered around the flat. As some of them are too far away from the main HA instance, I have configured a second HA instance, and I am using this component to sync all the info to the main HA instance.

As both instances could receive status updates from any sensor, I defined all the sensors on both instances, using the same names, with the intention of having a single entity per sensor, that can be updated from any HA instance. However, when I restart the remote instance, the main instance complains with these messages, and stops updating the entity:

```
Platform ble_monitor does not generate unique IDs. ID t_Monitor already exists - ignoring sensor.ble_temperature_monitor
Platform ble_monitor does not generate unique IDs. ID h_Monitor already exists - ignoring sensor.ble_humidity_monitor
Platform ble_monitor does not generate unique IDs. ID batt_Monitor already exists - ignoring sensor.ble_battery_monitor
```

I guess I can use separate names or a entity prefix, then add some automation to update one entity from the other, or perhaps use a template to grab the info from the last updated entity. But it would be much easier if this component could merge the info from entities with the same name.

Thanks!

**Comment by postlund** [comment]
In trying to figure out how this actually works, but I'm still not sure. Will have to experiment a bit more. Not sure of the implications when allowing something like this, that's also something that needs to be investigated further.

**Comment by eduperez** [comment]
I have move to a different solution, and this is no longer an issue for me. If I am the only one affected by this, the issue can be closed; I do not want to waste your time.

Many thanks!

**Comment by estebanbouza** [comment]
@eduperez having the same issue here. Can you share what was your different solution?

**Comment by eduperez** [comment]
I had a main Home Assistant instance on a RPi 3, and a secondary one on a RPi Zero. The Zero is no longer capable to boot recent versions of Home Assistant, so I had to abandon this idea.

I installed a bare Raspbian on the Zero, and a small Python program that listens for BLE messages and sends the info via MQTT to the main instance.

This program is based on https://github.com/JsBergbau/MiTemperature2
